ms2query/results_table.py

```diff
@@ -72,3 +77,74 @@ def add_ms2query_meta_score(self,
 
     def get_training_data(self) -> pd.DataFrame:
         return self.data.drop("inchikey", axis=1)
+
+    def create_dataframe_with_compound_classification(
+            self,
+            nr_of_top_spectra,
+            classifiers_file_name,
+            add_selection: Union[bool, List[str]] = False):
+        selected_results = \
+            self.data.iloc[:nr_of_top_spectra, :].copy()
```

Comment:
That reminds we that we should add tests to check if copies are done right.
I mostly mean: if we change any value in the ResultsTable it should not affect the sources.
(mostly a note to self I guess.. )